### DISCUSSION TITLE
PorousFlow: how to set correct BC for boreholes

### DISCUSSION THREAD

**Discussion by jack-fi** [idaholab/moose#25229] [Q&A Modules: General] [answered]
Dear community,
I am trying to set a TH model for a storage case using porous flow. I have read documentation like [this ](https://mooseframework.inl.gov/modules/porous_flow/ates.html)but still need help. I made two cylinders in my mesh which are supposed to represent boreholes. The outer surface of one of the cylinders is also chosen as the BC for temperature. The material properties of the cylinders i.e. boreholes are selected in a way to represent a column on water (density=1000, porosity=1, ...). When I ran the simulation, only the outer boundary of the hot borehole heated up. In the following, you see some plots. The slice shows the hot area around the hot borehole while I expected it to heat up inside the borehole. Curves also pass through the section. As it can be seen the temperature value drops inside the borehole. The pressure behaves more normally and is increased in the borehole. My MOOSE input file and mesh are stored in my git repository at https://github.com/jack-fi/input_for_moose 

![simu](https://github.com/idaholab/moose/assets/139582324/24ac07fd-19a5-4de0-8d77-9b1fc940a3a9)

I very much appreciate your help.

**Comment by GiudGiud** [comment, marked as answer]
Hello

it seems unconventional to have dirichletBCs inside the domain. You will have better control over the amount of heat introduced using Neumann BCs

have you tried running this longer? I don’t know how much heat moves in 20s in the ground 

Guillaume

**Reply by cpgr** [reply]
I suspect @GiudGiud is correct and 20s isn't long enough to heat it up. Pressure propagates much faster than heat in a porous media, so it looks like what you are doing will work if you run it longer.

One comment is that the physics inside a borehole isn't really included in the porous flow module, and this physics is a bit different to flow in porous media, so I'm not sure how relevant your results will be?

**Reply by jack-fi** [reply]
Dear @cpgr and @GiudGiud Thanks for your hints and help.
